packages/datagateway-common/package.json

```diff
@@ -75,7 +75,7 @@
     "prepare": "yarn tsc",
     "test:watch": "react-scripts test --env=jsdom --watch",
     "lint:js": "eslint --ext=tsx --ext=ts --ext=js --ext=jsx --fix ./src",
-    "tsc": "tsc -p tsconfig.lib.json",
+    "tsc": "tsc -p tsconfig.lib.json && cp -r src/images lib/",
```

Comment:
This was to allow for images on the new homepage to be referred to by source code in lib, although it may not be needed... I'm not sure how dg-common exports images actually. Sometimes plugin images don't load properly on my local machine, so this is something I need to look into.

Reply:
This additional cp command has now been removed as it is no longer necessary